tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
```

Comment:
I believe separating out `cover` is really beneficial as it shows us what is wrong with the code: whether it breaks existing tests or fails to maintain minimum coverage percentage level.